tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -102,6 +130,10 @@ teardown() {
 
     [ "${SNAPSHOTTER:-}" = "nydus" ] || skip "None snapshotter was found but this test requires one"
 
-    kubectl describe pod "$pod_name"
+    kubectl describe pods
     k8s_delete_all_pods_if_any_exists || true
+    kubectl delete pvc trusted-pvc || true
```

Comment:
Please run these `kubectl delete` with `--ignore-not-found` (and without `|| true`). For example, `kubectl delete --ignore-not-found pvc trusted-pvc`.

So avoiding nasty messages like:
```
2024-08-13T09:03:23.7388751Z # Error from server (NotFound): persistentvolumeclaims "trusted-pvc" not found
2024-08-13T09:03:23.7390598Z # Error from server (NotFound): persistentvolumes "trusted-block-pv" not found
2024-08-13T09:03:23.7392602Z # Error from server (NotFound): storageclasses.storage.k8s.io "local-storage" not found
```

ps: I know that many tests don't pass that parameter, but it's not always too late to learn new habits :)

Reply:
Good point. Thanks for your suggestion @wainersm !

Reply:
Fixed.